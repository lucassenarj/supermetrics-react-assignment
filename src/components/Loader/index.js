import React from "react";
import "./index.scss";

function Loader() {
  return (
    <div className="loader">
      <div className="loader__spinner"></div>
    </div>
  );
}

export default Loader;
