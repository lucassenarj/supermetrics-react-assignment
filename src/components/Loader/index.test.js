import React from "react";
import { render } from "@testing-library/react";
import Loader from "./";

describe("Components", () => {
  describe("Loader", () => {
    it("renders correctly", () => {
      const { container } = render(<Loader />);

      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
