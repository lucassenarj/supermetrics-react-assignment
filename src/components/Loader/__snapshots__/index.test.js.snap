// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Loader renders correctly 1`] = `
<div
  class="loader"
>
  <div
    class="loader__spinner"
  />
</div>
`;
